python/src/scipp/plotting/figure1d.py

```diff
@@ -194,7 +195,7 @@ def _preprocess_hist(self, name, vals):
         """
         x = vals["values"]["x"]
         y = vals["values"]["y"]
-        centers = 0.5 * (x[1:] + x[:-1])
+        centers = midpoint(x[:-1], x[1:])
```

Comment:
Right. So to be on the safe side, we would need an element-wise operation which checks which operand is smaller. But since Simon already mentioned that we should wait before making this function public, I will just go with keyword args because that is enough for plotting right now.